ob-pattern-2/templates/obbi-dashboard/wso2ob-pattern-2-obbi-dashboard-service.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
@kaumini update the license header year.

Reply:
Fixed with edf1bb86b4ea94c37158885b59e738d865e2347a